pkg/config/authorization/opa.go

```diff
@@ -81,3 +98,51 @@ func generatePolicyUID(policyName string, policyContent string, nonce int) strin
 	data := []byte(fmt.Sprint(nonce) + policyUIDHashSeparator + policyName + policyUIDHashSeparator + policyContent)
 	return fmt.Sprintf("%x", md5.Sum(data))
 }
+
+type ResponseOpaJson struct {
```

Comment:
No need to export these types if they're only used for unmarshalling the response.

```suggestion
type responseOpaJson struct {
```